frontend/components/frontpage/RecentActivity.vue

```diff
@@ -1,132 +1,140 @@
 <script setup lang="ts">
-import scale from "assets/images/icons/scale.png";
-import pencil from "assets/images/icons/pencil.png";
 import { Scale } from "lucide-vue-next";
 import { Pencil } from "lucide-vue-next";
-import RecentActivity from "~/components/frontpage/RecentActivity.vue";
 
 type Status = "DRAFT" | "SUBMITTED_TO_SUPERVISOR";
 type Proposal = {
-  refNumber: number; //refNumber doesn't show the _, waiting for backend to decide what it should be.
-  title: string;
-  type: string;
-  status: Status;
-  dateSubmitted: string;
-  lastEdited: string;
+    refNumber: number; //refNumber doesn't show the _, waiting for backend to decide what it should be.
+    title: string;
+    type: string;
+    status: Status;
+    dateSubmitted: string;
+    lastEdited: string;
 };
 type Roadmap = {
-  refNumber: number;
-  roadmapBody: string;
-  stepsCompleted: string;
+    refNumber: number;
+    roadmapBody: string;
+    stepsCompleted: string;
 };
 
 //mockdata
 const staticDraftProposal: Proposal = {
-  refNumber: 234324_2025,
-  title: "Voorstel Kattenbiologie",
-  type: "proposal_in_ethics",
-  status: "DRAFT",
-  dateSubmitted: "2025-01-24",
-  lastEdited: "2025-04-23",
+    refNumber: 234324_2025,
+    title: "Voorstel Kattenbiologie",
+    type: "proposal_in_ethics",
+    status: "DRAFT",
+    dateSubmitted: "2025-01-24",
+    lastEdited: "2025-04-23",
 };
 const staticSendToProposal: Proposal = {
-  refNumber: 4234_2023,
-  title: "hi mom I am on the internet",
-  type: "proposal_in_ethics",
-  status: "SUBMITTED_TO_SUPERVISOR",
-  dateSubmitted: "2025-01-24",
-  lastEdited: "2025-04-23",
+    refNumber: 4234_2023,
+    title: "Placeholder",
+    type: "proposal_in_ethics",
+    status: "SUBMITTED_TO_SUPERVISOR",
+    dateSubmitted: "2025-01-24",
+    lastEdited: "2025-04-23",
 };
 const staticRoadmap: Roadmap = {
-  refNumber: 3_2025,
-  roadmapBody: "body content of roadmap",
-  stepsCompleted: "1/6",
+    refNumber: 3_2025,
+    roadmapBody: "body content of roadmap",
+    stepsCompleted: "1/6",
 };
 
 const recentActivity: RecentActivity[] = [
-  staticDraftProposal,
-  staticSendToProposal,
-  staticRoadmap,
+    staticDraftProposal,
+    staticSendToProposal,
+    staticRoadmap,
 ];
+
 type RecentActivity = Proposal | Roadmap;
-const isProposal = (obj: Proposal | Roadmap): obj is Proposal => {
-  return obj.hasOwnProperty("status");
+
+const isProposal = (
+    recentActivity: Proposal | Roadmap,
```

Comment:
```suggestion
    recentActivity: RecentActivity,
```

(also for the next function)